.github/workflows/release-quickstart.yml

```diff
@@ -0,0 +1,133 @@
+
+      - name: Set Up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: amd64,arm64
+
+      - name: Set Up Docker BuildKit
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          # it is preferable to obtain the username from a var so that
+          # recurrences of the same string are not masked in CI output
+          username: ${{ vars.DOCKER_HUB_API_USER || secrets.DOCKER_HUB_API_USER }}
+          password: ${{ secrets.DOCKER_HUB_API_TOKEN }}
+
+      - name: Compute the Ziti Quickstart Version String
+        id: get_version
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        shell: bash
+        run: |
+          QUICKSTART_VERSION="$($(go env GOPATH)/bin/ziti-ci -q get-current-version)"
+          # drop the leading 'v', if any
+          QUICKSTART_VERSION=${QUICKSTART_VERSION#v}
+          if ! [[ "${QUICKSTART_VERSION}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            # fail the job because we could not obtain the current version from ziti-ci
+            echo "ERROR: QUICKSTART_VERSION=${QUICKSTART_VERSION} is not a semver"
+            exit 1
+          elif [[ "${GITHUB_REF}" =~ ^refs/heads/(release-v|main$) ]]; then
+            # Set output parameters for release branches
+            echo "DEBUG: QUICKSTART_VERSION=${QUICKSTART_VERSION}"
+            echo QUICKSTART_VERSION="${QUICKSTART_VERSION}" >> $GITHUB_OUTPUT
+          else
+            # Append short sha for non-Ziti-release refs to identify quickstart docker images
+            # shipped from release-next
+            QUICKSTART_VERSION="${QUICKSTART_VERSION}-$(git rev-parse --short ${GITHUB_SHA})"
+            echo "DEBUG: QUICKSTART_VERSION=${QUICKSTART_VERSION}"
+            echo QUICKSTART_VERSION="${QUICKSTART_VERSION}" >> $GITHUB_OUTPUT
+          fi
+
+          # configure the env var used by the quickstart's Dockerfile to download the correct version of ziti for the
+          # target architecture of each image build by trimming the hyphenated short sha suffix so that the preceding
+          # release version of the ziti executable is installed in the quickstart container image
+          ZITI_OVERRIDE_VERSION=${QUICKSTART_VERSION%-*}
+          echo "DEBUG: ZITI_OVERRIDE_VERSION=${ZITI_OVERRIDE_VERSION}"
+          echo ZITI_OVERRIDE_VERSION="${ZITI_OVERRIDE_VERSION}" >> $GITHUB_OUTPUT
+
+      # This container differs in that :latest is pushed from branch release-next in addition to main and hotfix
+      # branches like releasev1.2.x. See https://github.com/openziti/ziti/issues/898
+      - name: Configure Quickstart Container
+        env:
+          IMAGE_REPO: ${{ env.ZITI_QUICKSTART_IMAGE }}
+          IMAGE_TAG:  ${{ steps.get_version.outputs.QUICKSTART_VERSION }}
+        id: tagprep_qs
+        shell: bash
+        run: |
+          DOCKER_TAGS=""
```

Comment:
is this DOCKER_TAGS even needed?

Reply:
The purpose of this line is to ensure the value is an empty string prior to composing the value. This is an example of a Docker tag step that recurs in our CI, and I meant to preserve the tag step shape by way of convention. I don't feel it's strictly necessary in this particular case, but leaned toward consistency of the glaring explicitness of this approach.